modules/serverless-functions-podman.adoc

```diff
@@ -4,11 +4,13 @@
 
 :_content-type: PROCEDURE
 [id="serverless-functions-podman_{context}"]
-= Using podman
+= Setting up Podman
 
-If you are using podman, you must run the following commands before getting started with {FunctionsProductName}:
+To use advanced container management features, you might want to use Podman with {FunctionsProductName}. To do so, you need to start the Podman service and connect to it.
 
-. Start the podman service that serves the Docker API on a UNIX socket at `${XDG_RUNTIME_DIR}/podman/podman.sock`:
+.Procedure
+
+. Start the Podman service that serves the Docker API on a UNIX socket at `${XDG_RUNTIME_DIR}/podman/podman.sock`:
```

Comment:
FWIW this is not strictly required. The `func` binary will attempt to start podman if it's not currently running. However, this has been a bit buggy in the last release, so maybe best not to mention it at the moment.